src/Emulator/Visuals/Visuals.css

```diff
@@ -2,4 +2,66 @@
     background-color: darkgrey;
     height: 50vh;
     width: 75vw;
-}
\ No newline at end of file
+}
+
+.imageContainer {
+    position: relative;
+    overflow: auto;
+    padding: '0';
+    border-style: solid;
+    border-color: blue;
+    margin: auto;
+ }
+ 
+ .imageSpace {
+    height: 1000px;
+    width: 1000px;
+    padding: 10px;
+    display: flex;
+    text-align: center;
+    position: relative;
+ }
+ 
+ .image {
+    position: relative;
+    zoom: 1;
+    display: inline-block;
+    border-style:solid;
+    -webkit-user-drag: none;
+    -khtml-user-drag: none;
+    -moz-user-drag: none;
+    -o-user-drag: none;
+ }
+ 
+ .iconButtons {
+    background-color: transparent;
+    border-color: transparent;
+ }
+ 
+ .icons {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    /* left: 50vh;
+    bottom: 100px; */
```

Comment:
what's this?

Reply:
It was a comment I forgot to delete. It's removed now